### PR TITLE
Deprecated release

### PR DESCRIPTION
**Pull request by zrzka**
* Bump version to 0.4.0 as it contains breaking changes
* Mark the crate as deprecated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 # Crossterm Utilities
 
+**The `crossterm_utils` crate is deprecated and longer maintained. The GitHub repository will
+be archived soon. All the code is being moved to the `crossterm` crate. You can learn more in
```
Comment (TimonPost) [inline README.md]: Maybe a quick link to crossterm here.

File: src/lib.rs [hunk 2]
```diff
@@ -2,6 +2,11 @@
 
 //! # Utils
 //!
+//! **The `crossterm_utils` crate is deprecated and longer maintained. The GitHub repository will
+//! be archived soon. All the code is being moved to the `crossterm` crate. You can learn more in
```
Comment (TimonPost) [inline src/lib.rs]: and here

File: README.md (hunk 1 above)
Comment (zrzka) [reply README.md]: Done.

File: src/lib.rs (hunk 2 above)
Comment (zrzka) [reply src/lib.rs]: Done.

**Review by TimonPost — APPROVED** [review]

**Pull request merged**